tests/particle_test.cc

```diff
@@ -737,13 +765,16 @@ TEST_CASE("Particle is checked for 2D case", "[particle][2D]") {
     REQUIRE_NOTHROW(particle->compute_volume());
 
     // Compute mass
-    REQUIRE_NOTHROW(particle->compute_mass());
+    REQUIRE_NOTHROW(mpm::particle::compute_mass<Dim>(particle));
```

Comment:
There is no `try` in `compute_mass` actually. Should we still require this?

Reply:
Sorry, there are `try` within the functions in this function. If any would `throw`, it would be caught here right?

Reply:
yes